### DISCUSSION TITLE
Running folder doesnt run test

### DISCUSSION THREAD

**Discussion by teempek** [thunderclient/thunder-client-support#1333] [Q&A]
Hi,  I am trying to run folder with several request. Every request has test that saves part of the response to variable. Every request works and saves the variable if I run them separately, but if I run the folder it doesnt save the variables.

**Comment by rangav** [comment]
Hi @teempek 

Can you please confirm below
- Are you using free/paid version?
- Have you read the free version terms?: https://www.thunderclient.com/terms#free-plan

**Reply by teempek** [reply]
free version, yes have read the terms. Only one from our team thats using for now.

**Reply by rangav** [reply]
Thanks for the confirmation. 

Will verify the run folder and get back to you.

**Reply by rangav** [reply]
I just now tested it working for me

I think if you refresh the env view, you should see the updated data.

**Reply by teempek** [reply]
It doesnt work for me. Didnt help refresh. I have a subfolder that I am running. Could the problem be due to that?

**Reply by teempek** [reply]
It also doesnt update the time below the request on left panel

![image](https://github.com/rangav/thunder-client-support/assets/86219971/1655bb1a-e846-40d3-b143-4aad620cbcc1)

**Reply by rangav** [reply]
can you create a new collection and folder and use this test URL - https://www.thunderclient.com/welcome

and can try to reproduce the issue

if you have the issue, export and share this test collection, I will verify it

**Reply by rangav** [reply]
and the times displayed are last modified date for the request, not last run time

**Reply by teempek** [reply]
Dont know how to do this

**Reply by rangav** [reply]
what you don't know? sorry not clear?

**Reply by teempek** [reply]
How to use this test URL - https://www.thunderclient.com/welcome
